packages/react/time-series/src/TimeSeries.tsx

```diff
@@ -1,3 +1,4 @@
+import { TimeSeriesQuery, TimeSeriesQueryVariables } from '@propeldata/ui-kit-graphql/src/generated'
```

Comment:
is this import from `src` intentional?

Reply:
I think this is something we should fix and export directly from the package, thanks for catching it!

Reply:
WebStorm stuff. Will fix. 👍